tests/models/test_google.py

```diff
@@ -835,6 +835,18 @@ async def test_google_model_text_document_url_input(allow_model_requests: None,
     )
 
 
+async def test_google_model_json_document_url_input(allow_model_requests: None, google_provider: GoogleProvider):
+    m = GoogleModel('gemini-2.5-pro', provider=google_provider)
+    agent = Agent(m, system_prompt='You are a helpful chatbot.')
+
+    json_document_url = DocumentUrl(url='https://kamalscraping-collab.github.io/sample-data/sample_transcript.json')
```

Comment:
Can we please use a different public JSON file that's not dependent on your repo?